manifests/init.pp

```diff
@@ -56,17 +56,22 @@
 #
 class svckill (
   Array[String]               $ignore       = [],
+  Array[String]               $ignore_defaults = [],
   Array[Stdlib::Absolutepath] $ignore_files = [],
-  Boolean                     $verbose      = true
+  Boolean                     $verbose      = true,
```

Comment:
Does anyone know if there's a reason we don't also expose the `svckill` type's `mode` attribute in a class parameter like we do with `$verbose`?